hack/local-up-cluster.sh

```diff
@@ -202,6 +202,21 @@ EXPERIMENTAL_CRI=${EXPERIMENTAL_CRI:-"false"}
 # Ensure CERT_DIR is created for auto-generated crt/key and kubeconfig
 mkdir -p "${CERT_DIR}" &>/dev/null || sudo mkdir -p "${CERT_DIR}"
 CONTROLPLANE_SUDO=$(test -w "${CERT_DIR}" || echo "sudo -E")
+# If on an selinux system, need to set the correct context so cfssl container can access it
+which getenforce &>/dev/null
+if [ $? -eq 0 ]; then
+    if [ "$(getenforce)" == "Enforcing" ]; then
+        # Try newer container_file_t type first
+        ${CONTROLPLANE_SUDO} chcon -R -t container_file_t "${CERT_DIR}" &>/dev/null
```

Comment:
`&>/dev/null` smells bad -- if this breaks and someone is debugging, won't they want to see the error message?

If you only conditionally want to show the message,
```sh
if ! output="$( ${CONTROLPLANE_SUDO} chcon -R -t container_file_t "${CERT_DIR}" 2>&1 )"; then
    echo "something went wrong!"
    echo "${output}"
fi
```